qt/applications/workbench/workbench/plugins/test/test_memorywidget/test_memorypresenter.py

```diff
@@ -0,0 +1,50 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+#    This file is part of the mantid workbench.
+#
+#
+from workbench.plugins.memorywidget.memoryview import MemoryView, \
+    from_normal_to_critical, from_critical_to_normal
+from workbench.plugins.memorywidget.memorypresenter import MemoryPresenter
+
+import unittest
+from unittest import mock
+
+
+class MemoryPresenterTest(unittest.TestCase):
```

Comment:
Could I ask the need for this test as well as the one in `mantidqt`? Is it not a copy?

Reply:
Thanks Martyn. I overlooked removing these tests from workbench.